operator/docs/dev/security/results.golint.txt

```diff
@@ -1,4 +1,5 @@
-main.go:69:10: if block ends with a return statement, so drop this else and outdent its block
+main.go:44:7: exported const OCPControllerNamespace should have comment or be unexported
```

Comment:
why do we need golint result checked in

Reply:
`gosec` and `golint` outputs are stored in location `doc/dev/security` location. This is handled in [PR#60](https://github.com/madhuthorat/ibm-spectrum-scale-csi/pull/60). 

Are you asking why these files are checked in as part of current PR? `gosec` and `golint` is linked to `make` command so that once development is done and developer test the build using `make` command along with it code scan also takes place. This would ensure before a PR gets merged the code is atleast once scanned using `gosec` and `golint`.

Let me know your views on this.